frontend/src/lib/desktop/components/forms/NumberField.test.ts

```
@@ -0,0 +1,381 @@
+        label: 'Test Number',
+        value: 0,
+        onUpdate,
+      },
+    });
+
+    const input = screen.getByRole('spinbutton');
+
+    // Test infinity - should not call onUpdate
+    await fireEvent.input(input, { target: { value: 'Infinity' } });
+
+    expect(onUpdate).not.toHaveBeenCalledWith(Infinity);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)